.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,7 +26,7 @@ i.e.
 - **WooCommerce**: v4.3.0
 - **WordPress**: v5.4.2
 - **PHP version**: 7.3.3
-- **Omise plugin version**: Omise-WooCommerce 4.1 (optional, in case of submitting a new issue)
+- **Omise plugin version**: Omise-WooCommerce 4.3 (optional, in case of submitting a new issue)
```

Comment:
I'm not sure you should really worry about keeping this updated as it's just an example

Reply:
I guess, not needed to update every time while releasing.